### PR TITLE
Feat/dapp 696

### PR DESCRIPTION
**Pull request by leodhb**
# Description
 - News flow on the native dapp
 - News are locked when user has a ticket
 - On the first time, an onboarding article will appear
simulator_screenshot_07E3D413-72EF-46CF-B0A9-DB4A7CE4F1FD

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Dangerous changes
- [ ] Chore (change that does not effect how the code works, eg: change color)
- [ ] Tests

### CODE REVIEW

**Review by thaisribon — COMMENTED** [review]
I was just curious, so i looked, but i think is best that nick or yan review this one too

File: src/components/moleculars/CardCenterImageButton/styles.ts [hunk 1]
```diff
@@ -4,19 +4,18 @@ import { defaultBodySmRegular } from "styles/typography/default";
...
+    elevation: 5,
+    shadowOffset: { width: -2, height: 4 },
+    shadowOpacity: 0.2,
+    shadowRadius: 5,
+    borderRadius: theme.spacingNative(4),
   },
   containerImage: {
     position: "relative",
-    borderRadius: 12,
+    borderRadius: theme.spacingNative(4),
```
[the remark this answers is not in the document]
Comment (leodhb) [reply src/components/moleculars/CardCenterImageButton/styles.ts]: FIXED

File: src/components/moleculars/ArticleSection/assets/NewsPaper.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
```
[the remark this answers is not in the document]
Comment (leodhb) [reply src/components/moleculars/ArticleSection/assets/NewsPaper.tsx:1]: I just got these icons from the handoff and parsed to jsx. Can we get these icons another way?

[the remark this answers is not in the document]
Comment (thaisribon) [reply src/components/moleculars/ArticleSection/assets/NewsPaper.tsx:1]: Yeah, you can do like this: 
```
<Icon
      type="rounded"
      name="newspaper"
      size={16}
      color={theme.colors.neutral10}
/>
```

[the remark this answers is not in the document]
Comment (leodhb) [reply src/components/moleculars/ArticleSection/assets/NewsPaper.tsx:1]: It's amazing! Thank yoooou, @thaisribon <3

**Review by yangalli — APPROVED** [review]

**Pull request merged**